src/components/Header/index.js

```diff
@@ -1,20 +1,44 @@
+import { Link, useHistory } from "react-router-dom";
 import logoImg from "../../assets/images/store-logo.png";
 import shoppingCartImg from "../../assets/images/shopping-cart.png";
-import { Container } from "./styled";
+import { Search, Submit } from "../../styled";
+import { Container, Img, ShoppingCart, SearchForm } from "./styled";
+import { useState } from "react";
```

Comment:
The absolut imports should be at the top, and the relative ones can come later. This is more of a convention rather than an issue.